### PR TITLE
Tracking section and bit of index

### PR DESCRIPTION
**Pull request by igneel64**

### CODE REVIEW

File: content/tracking/tracking_3.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+| name | `string` | Name or title of the product. | ✘ |
+| category | `string` | Category the product belongs to. Use a consistent separator to express multiple levels. E.g. Woman/Shoes/Sneakers. | ✘ |
+| list_price | `number` | Recommended or list price of a product. | ✘ |
+| quantity | `number` | Quantity of the product taking part in the action. Used for Cart events. | ✘ |
+| size | `string` | Size of the product. E.g. XL, XS, M. | ✘ |
+| variant | `string` | Variant of the product. E.g. Red, Heavy, Leather. | ✘ |
+| brand | `string` | Brand of the product. | ✘ |
+| inventory_status | `string` | Inventory status of the product. E.g. in stock, out of stock, preorder, backorder. | ✘ |
+| position | `number` | Position the product was presented in a list of products. Used in Product List events. | ✘ |
+| creative_id | `string` | Identifier/Name/Url for the creative presented on a list or product view. | ✘ |
```
Comment (rlh1994) [inline content/tracking/tracking_3.md:114]: ❓ Do we need this on both this and the previous page?

File: content/tracking/tracking_3.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+#### `trackRemoveFromCart`
+
+To track a product view you can use the `trackRemoveFromCart` method with the following attributes:
+
+```ts
+/* {trackerName} is a placeholder for the initialized tracker on your page.  */
+
+window.snowplow("trackRemoveFromCart:{trackerName}", { products: Product[], total_value, currency, cart_id });
+```
```
Comment (rlh1994) [inline content/tracking/tracking_3.md:88]: 💡 Could we give a real example as we do in other pages?

File: content/tracking/tracking_4.md [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+|     quantity     | `number` |                      Quantity of the product taking part in the action. Used for Cart events.                      |    ✘     |
+|       size       | `string` |                                        Size of the product. E.g. XL, XS, M.                                        |    ✘     |
+|     variant      | `string` |                                 Variant of the product. E.g. Red, Heavy, Leather.                                  |    ✘     |
+|      brand       | `string` |                                               Brand of the product.                                                |    ✘     |
+| inventory_status | `string` |                 Inventory status of the product. E.g. in stock, out of stock, preorder, backorder.                 |    ✘     |
+|     position     | `number` |               Position the product was presented in a list of products. Used in Product List events.               |    ✘     |
+|   creative_id    | `string` |                     Identifier/Name/Url for the creative presented on a list or product view.                      |    ✘     |
```
Comment (rlh1994) [inline content/tracking/tracking_4.md:234]: ❓ Same q again

File: content/tracking/tracking_5.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+- **billing_postcode** - Billing address postcode.
+- **shipping_full_address** - Full shipping address.
+- **billing_full_address** - Full billing address.
+- **delivery_provider** - Can be used to discern delivery providers E.g. DHL, PostNL.
+- **delivery_method** - Store pickup, standard delivery, express delivery, international.
+- **coupon_code** - Coupon applied at checkout.
+- **account_type** - Selection of 'existing user' or 'guest checkout'.
+- **payment_method** - Any kind of payment method the user selected to proceed. E.g. Card, PayPal, Alipay.
+- **proof_of_payment** - Invoice or receipt.
+- **marketing_opt_in** - If opted in to marketing campaigns to the email address.
```
Comment (rlh1994) [inline content/tracking/tracking_5.md:60]: 💡 example?

File: content/tracking/tracking_6.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+| name | `string` | Name or title of the product. | ✘ |
+| category | `string` | Category the product belongs to. Use a consistent separator to express multiple levels. E.g. Woman/Shoes/Sneakers. | ✘ |
+| list_price | `number` | Recommended or list price of a product. | ✘ |
+| quantity | `number` | Quantity of the product taking part in the action. Used for Cart events. | ✘ |
+| size | `string` | Size of the product. E.g. XL, XS, M. | ✘ |
+| variant | `string` | Variant of the product. E.g. Red, Heavy, Leather. | ✘ |
+| brand | `string` | Brand of the product. | ✘ |
+| inventory_status | `string` | Inventory status of the product. E.g. in stock, out of stock, preorder, backorder. | ✘ |
+| position | `number` | Position the product was presented in a list of products. Used in Product List events. | ✘ |
+| creative_id | `string` | Identifier/Name/Url for the creative presented on a list or product view. | ✘ |
```
Comment (rlh1994) [inline content/tracking/tracking_6.md:96]: ❓

**Review by rlh1994 — COMMENTED** [review]
Just some questions/suggestions. 

Am I right in saying there is no demo data we are giving them and no enrich topic? We'll need to change the flowcharts if that is the case. Also with no demo data should we move the tracking section before modelling?

File: content/_index.md [hunk 6]
```diff
@@ -1,8 +1,95 @@
...
+2h :next steps, after visualize, 2m
+
+{{</mermaid >}}
+
+---
+
+#### Prerequisites
+
+**Tracking and Enrichment**
+
```
[the remark this answers is not in the document]
Comment (rlh1994) [reply content/_index.md:68]: The only thing with this is that the sample web data won't have any of the ecommerce tracking in it, so it won't work for this accelerator

File: content/_index.md [hunk 7]
```diff
@@ -1,8 +1,95 @@
...
+{{% /notice %}}
+
+#### What you will build
+
+An example Ecommerce Dashboard with Streamlit
+
+{{< tabs groupId="dashboard_images" >}}
+
+{{% tab name="Streamlit" %}}
+
```
[the remark this answers is not in the document]
Comment (rlh1994) [reply content/_index.md:90]: Yep, just waiting for @Jack-Keene to find a new one now the old one is no longer available

File: content/_index.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (igneel64) [reply content/_index.md:69]: Added it with a different text, focusing more in 'starting out collecting web data'.

File: content/visualization/visualization_1.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+#### **Step 3:** Run the Streamlit dashboard
+Run the command below to run the streamlit locally
+
+```bash
+streamlit run Dashboard.py
+```
+
+The dashboard contains a selection of common ecommerce visualizations including shopping behavior, top abandoned products, product list performance and checkout abandonment. This is a great starting point to build further analysis on top of. 
+
+!['Streamlit Dashboard Example'](images/streamlit_dashboard.png)
```
Comment (Craig-Mace) [inline content/visualization/visualization_1.md:62]: Image currently not appearing

File: config.toml [hunk 9]
```diff
@@ -1,6 +1,6 @@
-title = "{{ACCELERATOR TEMPLATE}}"
-baseURL = "https://docs.snowplow.io/accelerators/{{YOUR BASE URL}}/"
+title = "Ecommerce"
+baseURL = "https://docs.snowplow.io/accelerators/ecommerce/"
 
 [params]
-  description = "{{ACCELERATOR DESCRIPTION}}"
+  description = "Tutorial for setting up Snowplow ecommerce tracking for your online store"
   author = "{{AUTHOR NAMES}}"
```
Comment (Jack-Keene) [inline config.toml:6]:
```suggestion
  author = "Peter Perlepes, Emiel Verkade, Ryan Hill"
```

**Pull request merged**